src/track/taglib/trackmetadata_id3v2.cpp

```diff
@@ -83,24 +83,72 @@ inline QString frameToQString(
     return toQString(frame.toString());
 }
 
+bool logWarningIfUnsupportedID3v2Frame(const TagLib::ID3v2::Frame* pFrame) {
```

Comment:
I'm still conflicted about this naming. I don't think its super fitting. What do you think about `isFrameUnknownAndLogIfSo`. I guess its a matter of priority whether we care about the logging side-effect or the meaning of the return value.

Reply:
https://github.com/mixxxdj/mixxx/pull/4610/commits/33da3c9fe424533eb61a0324802e865f360e93d8